src/types/companies.ts

```diff
@@ -0,0 +1,9 @@
+export interface Companies {
+  [name: string]: number;
+}
+
+export interface Company {
+  id: string;
+  name: string;
+  persisted: boolean;
```

Comment:
What's the main reason to declare and send over the network the `persisted` property? It does not seem to be used anywhere in the front-end code.